webapp/app/plugins/pinboard.js

```diff
@@ -108,12 +108,28 @@ treeherder.controller('PinboardCtrl', [
         $scope.toggleEnterBugNumber = function(tf) {
             $scope.enteringBugNumber = tf;
             $scope.focusInput = tf;
+            if (tf) {
+                // We want escape to first cancel entering the bug number, so
+                // unbind the hotkey. Pressing escape again will close the
+                // panels as usual.
+                Mousetrap.unbind(['escape']);
```

Comment:
I'm pretty sure you don't need the `[]` for a single key, they are used only for lists of keys.